shared/constants/field/myinfo/myinfo-dialects.ts

```diff
@@ -1,4 +1,4 @@
-const myInfoDialects = [
+export const myInfoDialects = [
```

Comment:
If we are exporting all these, then we have no need to export default below right?

Reply:
oops yes! fixed in [ccae981](https://github.com/opengovsg/FormSG/pull/6694/commits/ccae981beabcb7d7de72daa26776b2e2bb38e9eb)